Framework/API/inc/MantidAPI/ParameterReference.h

```diff
@@ -44,6 +44,8 @@ namespace API {
 class MANTID_API_DLL ParameterReference {
 public:
   ParameterReference();
+  ParameterReference(const ParameterReference&){}
```

Comment:
What's the reason that we have a user-defined copy constructor that doesn't do anything along with an implicit copy-assignment operator?